etc/mediathekview.profile

```diff
@@ -23,6 +28,7 @@ protocol unix,inet,inet6
 seccomp
 tracelog
 
+disable-mnt
```

Comment:
@SpotComms It's for TV on demand, so everything is supposed to happen via Internet. Additionally, the default VLC closes automatically at the end of each film/movie. 

But still: It is of course not impossible to take a sandboxed, crippled, TV playing media player and try to access a local movie database. So yes, this change is debatable and has its con. It would make the profile more robust if we dropped disable-mnt. Maybe I was not abstracting enough from the way I use it :)